common/src/Assets/TextureBuffer.cpp

```diff
@@ -29,6 +29,26 @@
 
 namespace TrenchBroom {
     namespace Assets {
+        TextureBuffer::TextureBuffer() : m_buffer(), m_size(0) {}
+
+        /**
+         * Note, buffer is created defult-initialized (i.e., uninitialized) on purpose.
+         */
+        TextureBuffer::TextureBuffer(const size_t size) :
+        m_buffer(new unsigned char[size]),
```

Comment:
Can you use std::make_unique instead?

Reply:
No, that zeroes the memory. In c++20 they added std::make_unique_for_overwrite that does what I want here.

Reply:
Ah, thanks for clarifying.